install/sqls/00007-hidden-admins.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE group_coadmins ADD COLUMN hidden BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE groups ADD COLUMN owner_hidden BOOLEAN NOT NULL DEFAULT FALSE AFTER owner_comment;
```

Comment:
not sure why codacy's throwing an error like crazy lulz (https://app.codacy.com/gh/openvk/openvk/pullRequest?prid=8370800)

Reply:
uuh maybe you should write strings like <code>\`string\`</code>, not just `string`